src/screens/Notification.js

```diff
@@ -0,0 +1,314 @@
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.unknown.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    "status-unknown"
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.normal.icon">
+          <Description>
+            The default icon for a normal status Notification.
+          </Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<StatusGoodSmall />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.normal.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    "status-ok"
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.warning.icon">
+          <Description>
+            The default icon for a warning status Notification.
+          </Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<StatusWarningSmall />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.warning.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    "status-warning"
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.critical.icon">
+          <Description>
+            The default icon for a critical status Notification.
+          </Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<StatusCriticalSmall />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.critical.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    "status-critical"
+                `}
+            </Example>
```

Comment:
Same alignment comment as above. Show "status-critical" as the string example instead of "brand". "status-warning" doesn't need to be wrapped in {}. Only examples that take up multiple lines or have special characters need to be wrapped with {}. Also PropertyValue should have a type prop.